### PR TITLE
reformatting functions zeros

### PR DESCRIPTION
**Pull request by kirodev**
After reformatting functions, All frameworks tests passed 100%
Capture d’écran 2022-02-23 204035

### CODE REVIEW

**Comment by djl11** [comment]
Thanks!

**Comment by kirodev** [comment]
You're welcome ! :)

File: ivy/functional/backends/torch/array_api/creation_functions.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+# local
+from ivy import dtype_from_str, default_dtype, dev_from_str, default_device
+
+
+# noinspection PyShadowingName
+def zeros(shape: Union[int, Tuple[int, ...]],
+          dtype: torch.dtype = None,
+          device: torch.device = None) \
+        -> Tensor:
```
Comment (andy1122) [inline ivy/functional/backends/torch/array_api/creation_functions.py:14]: THis should be troch.Tensor....

Comment (kirodev) [reply ivy/functional/backends/torch/array_api/creation_functions.py:14]: I Imported Tensor from torch !

**Pull request merged**